integrations/extensions/starter-kits/watson-discovery/README.md

```diff
@@ -0,0 +1,114 @@
+
+## Limit on Size of Search Results
+
+If you look closely at the configurations in the actions that call the search extension ("Search", "Search within the mining topic", and "How many Bitcoins can exist?") you will notice that they set an assortment of parameters that impact how many bytes of data Watson Discovery returns.  Most notably, the `count` field is set to 3 (so we are asking for at most 3 search results) and the `return` field is set to `["title","metadata.source.url"]` so we are only getting the the title and URL from the document schema (in addition to the passages from the `text` field, because `passages.enabled` is `true` and `passages.fields` is `["text"]`).  If we omitted the `return` field, we would be getting back everything in the schema which in our demo example would include the full HTML of the FAQ page (which is fairly large).  In addition, we set `table_results.enabled` to `false` because we don't need tables in this application (but small tables may well be fine within the context of this limit).
+
+Watson Assistant extensions have a hard limit of 100kb on the size of search results, and if the result from your extension exceed that limit, the action will fail without any visible warning or error.  So be sure to configure your Watson Discovery query cautiously and avoid pulling in anything you do not actually need.  As long as you are only asking for text that could plausibly fit into a search result you would want to display in a conversational bot, this limit should not be a severe restriction -- 100kb is a very sizable amount of text.  However, if you configure Watson Discovery carelessly and drag in a lot of extra content you do not intend to use or display, then it is possible to hit this limit and cause your searches to fail.  If you are not sure whether your Watson Discovery queries will generate responses that are more than 100kb in size, try calling the Watson Discovery API directly using the API testing tool of your choice and measuring the size of the response.
+
```

Comment:
It could be helpful t show an example of what such an API call and its results would look like.